client/src/components/Reviews_Ratings/Ratings_Reviews.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
```

Comment:
I think for consistency, Steve and I are using the variable name token instead of API_KEY, but maybe API_KEY is more explicit. Maybe something to talk about in a future meeting.